rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -30,3 +30,171 @@ pkg_locations:
   - /etc/openstack_deploy
   - /opt/rpc-openstack/rpcd
 
+# TODO(odyssey4me):
```

Comment:
Yes, as the group_vars patch only recently merged I haven't had the chance to switch this config over. I've been focusing on trying to get this passing builds so that we can merge it to unblock the Ocata work. My intent will be to push up another PR on top of this to do that.